content/teams_intro.tex

```diff
@@ -81,8 +81,8 @@ \subsubsection*{Team Creation}
 is detailed further in Section~\ref{subsec:shmem_team_config_t}.
 
 \acp{PE} in a newly created teams are consecutively numbered with starting with
-\ac{PE} number 0. \acp{PE} are always ordered by the existing global \ac{PE} number that
-would be returned by the \FUNC{shmem\_my\_pe} routine. Team relative \ac{PE}
+\ac{PE} number 0. \acp{PE} are always ordered by the existing \ac{PE} number that
+would be returned by the \FUNC{shmem\_team\_my\_pe} routine called on the parent team. Team relative \ac{PE}
```

Comment:
could remove the word "existing".
Or possible reword:
"The \acp{PE} in a newly created team are always ordered by their \ac{PE} number in the parent team."

Reply:
Thanks!  Applied these changes in fe0ad7b.